Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.23-alpine3.20 AS builder
+
+ENV GO111MODULE=on
+
+WORKDIR /gemini
+
+COPY . .
+
+RUN apk add build-base musl-dev \
```

Comment:
Have you verified that usage of the `musl` doesn't have significant performance impact?
In rust case the diff is about 10 times.

Here is an example of building still a small docker image and without musl: https://github.com/scylladb/scylla-cluster-tests/blob/master/docker/latte/Dockerfile

Reply:
> Have you verified that usage of the `musl` doesn't have significant performance impact? In rust case the diff is about 10 times.

Yes, it does have performance impact, not 10 times, at 2 times at least, I've changed it to use glibc instead

Reply:
> Good point, let's use static stdlibc via `CGO_ENABLED=0` and get rid of env dependency.

Done